ui/info/info.go

```diff
@@ -28,15 +28,17 @@ type Model struct {
 	Err    error
 	User   *charm.User
 	cc     *client.Client
+	cfg    *client.Config
 	styles common.Styles
 }
 
 // NewModel returns a new Model in its initial state.
-func NewModel(cc *client.Client) Model {
+func NewModel(cc *client.Client, cfg *client.Config) Model {
 	return Model{
 		Quit:   false,
 		User:   nil,
 		cc:     cc,
+		cfg:    cfg,
```

Comment:
Nice work! You don't have to pass `*client.Config` here, the charm client `*client.Client` already has a reference to `*client.Config`.  You could use `m.cc.Config.Host` to get the host.

Reply:
Thanks so much! Just changed this. I tried `m.cc.cfg.Host` yesterday and was like "it must not exist" LOL late night coding bunni could be more observant 😂